db/glue.c

```diff
@@ -247,6 +247,13 @@ void init_fake_ireq_auxdb(struct dbenv *dbenv, struct ireq *iq, int auxdb)
     iq->use_handle = get_bdb_handle_ireq(iq, auxdb);
 }
 
+struct ireq *get_fake_ireq()
+{
+    struct ireq *iq = malloc(sizeof(struct ireq));
+    init_fake_ireq(thedb, iq);
```

Comment:
for extra safety, calloc() instead?

Reply:
sure

Reply:
eh - `init_fake_ireq` is a constructor and already bzeros

Reply:
It does appear to bzero portions of the ireq; however, since it does not bzero the entire thing, it's difficult to verify at a glance that the struct will be in a well-defined initial state upon return.